modules/ROOT/pages/salesforce/salesforce-connector-reference-100.adoc

```diff
@@ -0,0 +1,4950 @@
+| Time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Read Timeout |  |
+| Target Variable a| String |  The name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of <<FindDuplicatesResult>>
+|===
+
+=== For Configurations
+* <<sfdc-config>>
+
+==== Throws
+* SALESFORCE:LIMIT_EXCEEDED
+* SALESFORCE:CONNECTIVITY
+* SALESFORCE:INVALID_RESPONSE
+* SALESFORCE:RETRY_EXHAUSTED
+* SALESFORCE:TIMEOUT
+* SALESFORCE:MUTUAL_AUTHENTICATION_FAILED
+* SALESFORCE:NOT_FOUND
+* SALESFORCE:INVALID_INPUT
+
+
+[[getAllJobsBulkApiV2]]
+=== Get All Jobs Bulk Api V2
+`<salesforce:get-all-jobs-bulk-api-v2>`
+
+
+Retrieves all Bulk Jobs  This call uses the Bulk API v2.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Concurrency Mode a| Enumeration, one of:
+
+** Parallel
+** Serial |  The desired concurrency mode. |  Parallel |
+| Pk Chunking a| Boolean |  Use the PK Chunking request header to enable automatic primary key (PK) chunking for a bulk query job. |  false |
+| Read Timeout a| Number |  Specifies the amount of time, in the unit defined in Time unit, that the consumer waits for a response before it times out. |  |
+| Time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Read Timeout |  |
+| Target Variable a| String |  The name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of <<BulkJobV2Result>>
+|===
+
+=== For Configurations
+* <<sfdc-config>>
+
+==== Throws
+* SALESFORCE:LIMIT_EXCEEDED
+* SALESFORCE:CONNECTIVITY
+* SALESFORCE:INVALID_RESPONSE
+* SALESFORCE:RETRY_EXHAUSTED
+* SALESFORCE:TIMEOUT
+* SALESFORCE:MUTUAL_AUTHENTICATION_FAILED
+* SALESFORCE:NOT_FOUND
+* SALESFORCE:INVALID_INPUT
+
+
+[[getAllQueryJobsBulkApiV2]]
+=== Get All Query Jobs Bulk Api V2
+`<salesforce:get-all-query-jobs-bulk-api-v2>`
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value |
```

Comment:
```suggestion
Deletes all of the access token information of a given resource owner ID so that it's impossible to execute an operation for that user without performing the OAuth dance.
```